lib/rules/no-multiline-var-declaration.js

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
Please run `grunt standards` on this branch as described in https://github.com/silvermine/eslint-plugin-silvermine/pull/3#discussion_r76612772 and fix the issues.